### ISSUE TITLE
Internet Explorer

### ISSUE THREAD

**Issue by angie-graphics** [dfringeli/ktf2021#25]
Hallo Dominik
Ich habe mir heute die Webseite im IE angeschaut, da du es gestern erwähnt hattest.

Was aus meiner Sicht zwingend behoben werden müsste sind folgende 2 Punkte:

1. Palme auf der Startseite
![image](https://user-images.githubusercontent.com/46542480/62190080-cab3c980-b370-11e9-923a-79278329ded4.png)

2. Die Navigation, die gar nicht aufklappt > nicht bedienbar in IE

Bei den News konnte ich kein Problem sehen. Aber ich denke oben stehende Punkte sind lösbar für die IE mittels browser-hack

**Comment by angie-graphics** [comment]
Navi: dasselbe gilt für die Suche, ist aber glaube ich der gleiche Fehler:

![image](https://user-images.githubusercontent.com/46542480/62190179-06e72a00-b371-11e9-9bf9-83a09879f3ec.png)

**Comment by dfringeli** [comment]
Ich hab die Palme im IE entfernt.

Und die Navigation und die Suche funktionieren

**Comment by angie-graphics** [comment]
heisst das, die palme ist auch hier ganz weg wie im safari? siehe https://github.com/dfringeli/ktf2021/issues/28 > die stilelemente weglassen ist nicht die richtige lösung. bitte nochmals prüfen :-)

**Comment by dfringeli** [comment]
Ja - lieber keine Palme als so:
![image](https://user-images.githubusercontent.com/33898687/78421544-bc812000-7658-11ea-93c8-b56d97a3702a.png)

Aber ich schau es nochmals an

**Comment by dfringeli** [comment]
ahh jetzt fällt es mir wieder ein.

der Internet Explorer hat nicht nur das gleiche Problem wie Safari, sondern zusätzlich noch, dass der IE flex nicht kann...

**Comment by angie-graphics** [comment]
ja, aber wenn man im webbereich immer alles weglassen würde, was probleme macht, dann würden die seiten ja kaum mehr elemente enthalten. ansonsten soll marco kohler mal suchen, damit du nicht mehr zeit investieren musst. er wollte ja bei der webseite ursprünglich helfen

**Comment by dfringeli** [comment]
Dein Kommentar ist weder konstruktiv noch lösungsorientiert. Ich habe gesagt, ich schaus mir nochmals an, aber ich bin kein Browser-Kompatibilität-Spezialist und kann nicht etliche Stunden investieren - muss ich auch nicht.

Ja, ich zeige ihm die Aufgabe.

**Comment by angie-graphics** [comment]
sorry, aber das ist im web einfach so.. da trifft man non-stop auf diese probleme und muss dann eben suchen, um sie zu lösen. das ist doch das, was den reiz ausmacht.. sobald ein hindernis kommt, wil man es doch lösen.. 

klar, hast du etliche stunden investiert, wir sind auch sehr dankbar dafür.. jedoch haben wir anderen auch etliche stunden investiert, ohne eine gegenleistung. es geht ja darum, dass man ein tolles fest haben möchte und motiviert ist, dieses ziel zu erreichen.. deshalb meldet man sich doch freiwillig.. dachte ich zumindest. 

mit kritik muss man umgehen können.. wenn du wüsstest wie häufig ich logos & co. machen musste bis mal eines allen gefallen hat. das gehört aber dazu und mir persönlich macht es auch spass und kritik spornt mich an, es besser zu machen, auch wenn ich gerne früher aufgeben würde. ;-) also sei nicht gleich beleidigt, wegen meines kommentars. :-) 

aber eben.. wenn du keine zeit oder lust mehr hast, dann kannst du das jederzeit melden und wir suchen ersatz.

**Comment by dfringeli** [comment]
Ich bleib bei meiner Meinung, dass dein kommentar auch keine kritik, sondern eben ein unkonstruktiver kommentar ist.

Aber du hast schon recht. Meine reaktion ist ein zeichen, dass ich nicht mit 100 dabei bin. Nicht weil ich keine lust mehr hab, sondern keine zeit.

Ich überlege mir das die nächsten Tage, ob ich meine ämtli fürs ktf niederlegen soll - oder einfach bis oktober, nach dem studium.

**Comment by angie-graphics** [comment]
das ist schon ok für mich, dass du meinen kommentar nicht als kritik ansiehst. ich bleibe auch dabei, dass dein kommentar mit "lieber keine palme als so" eine echt merkwürdige herangehensweise ist..ein web-problem zu lösen. wenn ich sowas bei der arbeit einem kunden sagen würde, wäre der gleich bei der nächsten web-agentur. 

ja, wenn du keine zeit hast, dann musst du das melden, dann kann man ersatz suchen. 
ich habe übrigens auch nicht gesagt, dass es morgen erledigt sein soll.. sondern nur, dass man das so nicht stehen lassen kann bis zum fest. wer es schlussendlich fixt, ist mir persönlich egal.

genau, schlaf mal drüber, komm wieder etwas runter und dann entscheide aus dem bauch :-)
